src/EditorFeatures/VisualBasicTest/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.vb

```diff
@@ -484,10 +481,7 @@ End Class",
     End Function
 
     Public Overrides Function GetHashCode() As Integer
-        Dim hashCode As Long = 2118541809
-        hashCode = (hashCode * -1521134295 + a.GetHashCode()).GetHashCode()
-        hashCode = (hashCode * -1521134295 + b.GetHashCode()).GetHashCode()
-        Return hashCode
```

Comment:
i'd revert these, and then actually pass in an appropriate option where tuples aren't available.    this code is actually attempting to test this "use a Long for the hashcode" approach :-/

Reply:
or keep your change, but add a test that validates what we generate when we don't have tuples available.